src/moj/components/date-picker/_date-picker.scss

```diff
@@ -0,0 +1,278 @@
+.moj-datepicker {
+  position: relative;
+
+  &--fixed-width {
+    .moj-datepicker-input__wrapper {
+      width: 215px;
+    }
+  }
+
+  &__dialog {
```

Comment:
We should use `@include govuk-font(16);` on this to make sure it has the right font face and size